components/layout/TopNav.tsx

```
@@ -0,0 +1,144 @@
+    const url = new URL(window.location.href)
+    url.searchParams.set('cluster', label)
+    window.history.replaceState(null,'',url.toString())
+  }
+
+  const connected = !!wallet.publicKey
+  const { wallets: myWallets } = useMyWallets()
+  const navSelect = (section: string) => track({ name:'navigation_select', section })
+  const onWalletPage = !!router.query.walletId
+
+  return (
+    <nav className="fixed top-0 left-0 right-0 h-14 border-b border-[var(--border-subtle)] bg-[var(--glass-bg)]/50 backdrop-blur-md flex items-center px-4 gap-4 z-40" role="navigation" aria-label="Primary">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<nav>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)